pkg/systemd/logs.js

```diff
@@ -28,6 +28,71 @@ $(function() {
     cockpit.translate();
     var _ = cockpit.gettext;
 
+    var problems_client = cockpit.dbus('org.freedesktop.problems');
+    var service = problems_client.proxy('org.freedesktop.Problems2', '/org/freedesktop/Problems2');
+    var problems = problems_client.proxies('org.freedesktop.Problems2.Entry', '/org/freedesktop/Problems2/Entry');
+
+    // A map of ABRT's problems items and it's callback for rendering
+    var problem_render_callbacks = {'core_backtrace': render_backtrace,
+                                    'os_info': render_table_eq,
+                                    'environ': render_table_eq,
+                                    'limits': render_limits,
+                                    'cgroup': render_cgroup,
+                                    'namespaces': render_table_co,
+                                    'maps': render_maps,
+                                    'dso_list': render_dso_list,
+                                    'mountinfo': render_mountinfo,
+                                    'proc_pid_status': render_table_co,
+                                    'open_fds': render_open_fds,
+                                    'var_log_messages': render_multiline,
+                                    'not-reportable': render_multiline,
+                                    'exploitable': render_multiline,
+                                    'suspend_stats': render_table_co,
+                                    'dmesg': render_multiline,
+                                    'container_rootfs': render_multiline,
+                                    'docker_inspect': render_multiline
+                                    };
+
+    var problem_info_1 = ['reason', 'cmdline', 'executable', 'package', 'component',
+                          'crash_function', 'pid', 'pwd', 'hostname', 'count',
+                          'type', 'analyzer', 'rootdir', 'duphash', 'exception_type',
+                          'container', 'container_uuid', 'container_cmdline',
+                          'container_id', 'container_image' ];
+
+    var problem_info_2 = ['Directory', 'username', 'abrt_version', 'architecture', 'global_pid', 'kernel',
```

Comment:
`lastr_occurence` and `time` are rendered as integers.  We should render them as date/time strings.